.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "liveServer.settings.port": 5501
+    "liveServer.settings.port": 5502
```

Comment:
it was change by mistake the port is 5501. i'm very sorry for inconvenience . don't merge this one the other one index.html is totally correct.